collections/bitroomgrid.rs

```diff
@@ -0,0 +1,248 @@
+//! Dependencies: `serde`
+//!
+//! A 50×50 binary grid meant to be used to flag tiles in a room
+//!
+//! Has basic compression built in too for more compact serialization
+
+
+use serde::de::{self, Deserialize, Deserializer, Visitor};
+use serde::ser::{self, Serialize, Serializer};
+use std::fmt::{self, Write};
+
+const ROWS: usize = 10;
+const ROW_SIZE: usize = 32 * 8; // Each row holds 32 * 8 = 256 bits of data
```

Comment:
What does this represent? Most of the time,  you use either `ROW_SIZE / 8` or `ROW_SIZE * 8`.